src/Symfony/Bridge/Doctrine/Tests/PropertyInfo/DoctrineExtractorTest.php

```diff
@@ -62,6 +62,21 @@ public function testGetProperties()
         );
     }
 
+    public function testGetEmbeddableProperties()
+    {
+        if (!class_exists('Doctrine\ORM\Mapping\Embedded')) {
+            return;
```

Comment:
You should put `$this->markTestSkipped()` with message that embedded is not available.